templates/shortcode-helper.php

```diff
@@ -143,12 +143,22 @@
                 echo '<div class="list-item">';
                 echo "<h3>$title</h3>";
                 foreach ($collection as $shortcode) {
-                    echo do_shortcode($shortcode);
+                  $sc_search = array(
+                    'leaflet-map ',
+                    'leaflet-image ',
+                    'leaflet-wms ',
+                  );
+                  $replace   = array(
+                    'leaflet-map height=250 ',
+                    'leaflet-image height=250 ',
+                    'leaflet-wms height=250 ',
```

Comment:
Might not be the best way to do this, but I guess it works. I would have thought we could set height via css and make the css "!important".

Reply:
> I would have thought we could set height via css and make the css "!important".

Thank you for hint.